### PR TITLE
FW-2535 Show different faqs in different pages based on categories [Important] 

### PR DESCRIPTION
**Pull request by Prochnost55**
### What do you want to achieve?
- Add a setting so that user can define category to show on which page

### PR Checklist

- [x] I have tested it locally and all functionalities are working fine.
- [x] I have compared it with mocks and all design elements are the same.
- [ ] I have tested it in IE Browser.

### How was the code tested?

- append getFaqByCategory to kommunicate settings with the name of faq category 

### What new thing you came across while writing this code? 
-

### In case you fixed a bug then please describe the root cause of it? 
-

NOTE: Make sure you're comparing your branch with the correct base branch

### CODE REVIEW

File: webplugin/js/app/km-post-initialization.js [hunk 1]
```diff
@@ -7,13 +7,17 @@
 Kommunicate.postPluginInitialization = function (err, data) {
     // get the third party settings
     KommunicateKB.init(Kommunicate.getBaseUrl());
-    Kommunicate.getFaqList(data);
+    var categoryName;
+    if (kommunicate && kommunicate._globals && kommunicate._globals.getFaqByCategory) {
```
Comment (gargshantnu) [inline webplugin/js/app/km-post-initialization.js:11]: @devashishmamgain is this parameter name is okay `getFaqByCategory` ?

Comment (devashishmamgain) [reply webplugin/js/app/km-post-initialization.js:11]: no, this looks like a function name

Comment (gargshantnu) [reply webplugin/js/app/km-post-initialization.js:11]: @Prochnost55, please check this and update parameter name accordingly

File: webplugin/knowledgebase/kb.js [hunk 2]
```diff
@@ -4,6 +4,7 @@
         var KommunicateKB = {};
         var KM_API_URL = 'https://api.kommunicate.io';
         var KB_URL = '/kb/search?appId=:appId';
+        var KB_V2_URL = '/kb?categoryName=:categoryName&applicationId=:appId&status=published&type=faq'
```
[the remark this answers is not in the document]
Comment (Prochnost55) [reply webplugin/knowledgebase/kb.js:7]: Actually I later found out after digging more that we can use kb url to filter  by category name.
I will update the code with correct nomenclature.

File: webplugin/js/app/km-post-initialization.js (hunk 1 above)
Comment (Prochnost55) [reply webplugin/js/app/km-post-initialization.js:11]: ok i have changed it to faqCategory

**Review by gargshantnu — APPROVED** [review]

**Review by devashishmamgain — APPROVED** [review]

**Pull request merged**